bin/ocaml/doc.ml

```diff
@@ -0,0 +1,70 @@
+open Stdune
+open Import
+
+let doc = "Build and view the documentation of an OCaml project"
+
+let man =
+  [ `S "DESCRIPTION"
+  ; `P
+      {|$(b,dune ocaml doc) builds and then opens the documention of an OCaml project in the users default browser.|}
+  ; `Blocks Common.help_secs
+  ]
+
+let info = Cmd.info "doc" ~doc ~man
+
+let term =
+  let+ common = Common.term in
+  let config = Common.init common in
+  let request (setup : Import.Main.build_system) =
+    let dir = Path.(relative root) (Common.prefix_target common ".") in
+    let open Action_builder.O in
+    let+ () =
+      Alias.in_dir
+        ~name:(Dune_engine.Alias.Name.of_string "doc")
```

Comment:
Can you add that as `Dune_engine.Alias.doc`? (and change the of_string "doc" in that file to use it)

Reply:
There is already an Alias.doc variable, but it has the wrong type.

Reply:
This has been moved since then, you can find it in the `Alias0` module.

Reply:
You don't need to change the `Alias` module. This identifier already exists, it is in `Alias0` module https://github.com/ocaml/dune/blob/main/src/dune_rules/alias0.ml